homeassistant/components/zha/__init__.py

```diff
@@ -90,8 +93,15 @@
     """
 
     zha_data = hass.data.setdefault(DATA_ZHA, {})
+    zha_data[DATA_ZHA_PLATFORM_LOADED] = {}
     config = zha_data.get(DATA_ZHA_CONFIG, {})
 
+    zha_data[DATA_ZHA_DISPATCHERS] = []
+    for component in COMPONENTS:
+        zha_data[component] = []
+        coro = hass.config_entries.async_forward_entry_setup(config_entry, component)
+        zha_data[DATA_ZHA_PLATFORM_LOADED][component] = hass.loop.create_task(coro)
```

Comment:
Let's use async_create_task so we make sure it's awaited before we finis the startup phase.

```suggestion
        zha_data[DATA_ZHA_PLATFORM_LOADED][component] = hass.async_create_task(coro)
```

Reply:
I think line 131 does this for us: `hass.async_create_task(async_load_entities(hass, config_entry))` then inside of `async_load_entities` there is a gather that contains all of the tasks stored above in `zha_data[DATA_ZHA_PLATFORM_LOADED][component]`

Reply:
yep, awaiting for the tasks later.